util/Util.js

```diff
@@ -63,10 +63,97 @@ module.exports = class Util {
         return text.length > maxLen ? `${text.substr(0, maxLen - 3)}...` : text;
     }
 
+    static garfieldDay(month, year) {
+        function leapYear(x) {
+            return (x % 4 === 0 && x % 100 !== 0) || x % 400 === 0;
+        }
+
+        let day = 0;
+        if (
+            month === 1 ||
+            month === 3 ||
+            month === 5 ||
+            month === 7 ||
+            month === 8 ||
+            month === 10 ||
+            month === 12
+        ) {
+            day =
+                Math.floor(
+                    Math.random() * (Math.floor(31) - Math.ceil(1) + 1)
+                ) + Math.ceil(1);
+        } else if (month === 2) {
+            if (leapYear(year) === true) {
+                day =
+                    Math.floor(
+                        Math.random() * (Math.floor(29) - Math.ceil(1) + 1)
+                    ) + Math.ceil(1);
+            } else {
+                day =
+                    Math.floor(
+                        Math.random() * (Math.floor(28) - Math.ceil(1) + 1)
+                    ) + Math.ceil(1);
+            }
+        } else {
+            day =
+                Math.floor(
+                    Math.random() * (Math.floor(30) - Math.ceil(1) + 1)
+                ) + Math.ceil(1);
+        }
+        return day;
+    }
+
+    static garfieldURL(day, month, year) {
```

Comment:
I might move this into config.js. That is where I like to put static URLs

Reply:
I wouldn't do that, this whole thing literally just generates a URL. I am actually going to move it from util,js back into garfield.js since it is only used with that.